src/ws-header/ws-header.js

```diff
@@ -288,13 +289,14 @@ export class WSHeader extends Component {
           className="level-2"
           onMouseEnter={() => this.enterLevel2()}
           onMouseLeave={() => this.leaveLevel2()}
+          onClick={() => this.leaveLevel2()}
           ref={element => { this.level2 = element; }}
         >
           {this.props.links.map((parent, index) =>
             parent.children && parent.children.length &&
             <ul className="main-sub-menu" key={`sub-menu${index}`} ref={element => { this.subMenus[index] = element; }}>
               {parent.children.map((child, childIndex) =>
-                <li key={`sub-link-${index}-${childIndex}`}>
+                <li key={`sub-link-${index}-${childIndex}`} className={(child.isCurrent) ? 'is-current' : null}>
```

Comment:
This just about the naming. The menu item is active and we use is-active everywhere as active state.